pkgs/development/python-modules/pytest-testmon/default.nix

```
@@ -0,0 +1,32 @@
+{ pkgs
+, lib
+, pythonPackages
```

```suggestion
, pytest
, coverage
```

You need to add them a function arguments like this

---

I'm glad you didn't mention it before. I just figured it out independently. I love when there's no magic.